pkg/console/operator/sync_v400.go

```diff
@@ -301,6 +301,22 @@ func SyncConfigMap(co *consoleOperator, recorder events.Recorder, operatorConfig
 		return nil, false, mcErr
 	}
 
+	// Check if a custom logo is set on the operator config
```

Comment:
The problem here is that our `watches` are on everything but this resource.  So if the user edits the `console operator config` first, then creates the `configmap` after, it will be too late, the logo configmap won't get picked up.  We will have a timing issue.

We need to make sure we don't miss the update.  That means we need to setup a watch.

In `starter.go` we can use the `resourceSyncer` (since we know for certain we need this resource copied into our namespace) to accomplish the watch & the copy:

```golang
     resourceSyncerInformers := v1helpers.NewKubeInformersForNamespaces(
          // just the configmap, in the config namespace
     )

     resourceSyncer := resourcesynccontroller.NewResourceSyncController(
          // see:
          // https://github.com/openshift/cluster-authentication-operator/blob/master/pkg/operator2/starter.go#L89
          // if you need an example for setting this up 
     )

    if err := resourceSyncer.SyncConfigMap(
        // sync it into the console namespace
        // we are already watching configmaps in the console namespace, so our sync loop will kick when 
        // it picks up changes
    )
```

Reply:
The resourceSyncer requires the secret getter to be set. If that is reasonable, then I can go down this path.  I am looking at the kubeInformer logic, which is what I used for the configmaps in openshift-config-manged.

Reply:
I created a solution using the `informer`.  Check it out and we can discuss if needed.